client/app/bundles/course/assessment/question/programming/components/common/Editor.tsx

```diff
@@ -0,0 +1,38 @@
+import { ComponentProps } from 'react';
+import AceEditor from 'react-ace';
+import { LanguageMode } from 'types/course/assessment/question/programming';
+
+import 'ace-builds/src-noconflict/theme-github';
+
+interface EditorProps extends ComponentProps<typeof AceEditor> {
+  language: LanguageMode;
+  value?: string;
+  onChange?: (value: string) => void;
+  disabled?: boolean;
+}
+
+const Editor = (props: EditorProps): JSX.Element => {
+  const { language, value, disabled, onChange, ...otherProps } = props;
+
+  return (
+    <AceEditor
```

Comment:
Do we want to keep the old monokai theme? Also, include the max and min lines?

Reply:
I don't think we want to keep Monokai. I set it to GitHub's theme to keep it consistent with the submission page's code editor. If our tree-shaking works well, this should also reduce the imports on production builds, since we're only importing `theme-github`.

Reply:
Correction, if I add back the `maxLines` and `minLines` props, then the editor's height will be initially set to `minLines` (was 10 lines), then every new line will increase the editor's height. This is what's happening in production.

I removed these props to make it such that, the editor's height is some pixels (default from Ace Editor), and that height is fixed. If the number of lines exceed the height, then the editor scrolls internally—which is how it's supposed to have been.